pymeasure/instruments/instrument.py

```diff
@@ -244,36 +249,87 @@ def id(self):
             raise NotImplementedError("Non SCPI instruments require implementation in subclasses")
 
     # Wrapper functions for the Adapter object
-    def ask(self, command):
-        """ Writes the command to the instrument through the adapter
-        and returns the read response.
-
-        :param command: command string to be sent to the instrument
-        """
-        return self.adapter.ask(command)
-
     def write(self, command):
         """ Writes the command to the instrument through the adapter.
 
         :param command: command string to be sent to the instrument
         """
         self.adapter.write(command)
 
+    def write_bytes(self, content):
+        """Write raw bytes to the instrument."""
+        self.adapter.write_bytes(content)
+
     def read(self):
         """ Reads from the instrument through the adapter and returns the
         response.
         """
         return self.adapter.read()
 
-    def values(self, command, **kwargs):
-        """ Reads a set of values from the instrument through the adapter,
-        passing on any key-word arguments.
-        """
-        return self.adapter.values(command, **kwargs)
+    def read_bytes(self, count):
+        """Read `count` number of bytes."""
+        return self.adapter.read_bytes(count)
 
-    def binary_values(self, command, header_bytes=0, dtype=np.float32):
-        return self.adapter.binary_values(command, header_bytes, dtype)
+    # Communication functions
+    def ask(self, command, delay=0):
+        """ Writes the command to the instrument through the adapter
```

Comment:
```suggestion
        """ Write the command to the instrument through the adapter
```

Reply:
I know. I did not write the docstring, but kept the existing one (probably moved from somewhere, therefore marked as 'new'). Changing that much, we can correct the docstring as well.